.travis.yml

```diff
@@ -1,38 +1,24 @@
-# Sample .travis.yml for R projects.
 #
-# See README.md for instructions, or for more configuration options,
-# see the wiki:
-#   https://github.com/craigcitro/r-travis/wiki
-# 
-# Hack for 'error on warning' except when the warning is about vignette pdfs/html
-# Thanks to csgillespie for https://github.com/csgillespie/travis-examples/tree/master/bioconductor
-language: c
-sudo: required
+# Travis configuration guide for R language:
+#   https://docs.travis-ci.com/user/languages/r/
+#   https://github.com/craigcitro/r-travis/wiki/Porting-to-native-R-support-in-Travis
+#
+language: r
 dist: trusty
+sudo: required
+cache: packages
+warnings_are_errors: true
 
-before_install:
-  - curl -OL http://raw.github.com/craigcitro/r-travis/master/scripts/travis-tool.sh
-  - chmod 755 ./travis-tool.sh
-  - sed "s/grep -q WARNING/grep WARNING \| grep -v inst\/doc \| grep -v 'Status:.*WARNING'/" travis-tool.sh > ./tmp.sh
-  - mv tmp.sh travis-tool.sh
-  - chmod 755 ./travis-tool.sh  
-  - ./travis-tool.sh bootstrap
-install:
-  - ./travis-tool.sh bioc_install IRanges
-  - ./travis-tool.sh bioc_install GenomicRanges
-  - ./travis-tool.sh install_deps
-  
-script:
-  - echo "Commit:" $TRAVIS_COMMIT >> ./DESCRIPTION
-  - ./travis-tool.sh run_tests
+bioc_packages:
+  - IRanges
+  - GenomicRanges
 
-after_failure:
-  - ./travis-tool.sh dump_logs
+r_github_packages:
```

Comment:
We might not need to install them from github, CRAN versions were most likely updated till now.